restaurant_bill/src/containers/wrapper.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import calculateBill from '../scripts/calculate'
+
+class Wrapper extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      people: 2,
+      totalBill: 0,
+      extraTip: 0,
+    }
+  }
+
+  handleChange({ target: { name, value } }) {
+    if (value === "" && name === "people"){
+      this.setState({ [name]: 2 });
+    } else if(value === ""){
+      this.setState({ [name]: 0 });
+    } else {
+      this.setState({ [name]: value });
+    }
+  }
```

Comment:
comments for each `if` block would be very, very useful here. it's not immediately obvious what's going on.